corehq/apps/reports_core/templates/reports_core/base_template_new.html

```diff
@@ -59,11 +59,17 @@
         };
 
         var paginationNotice = function (data) {
-            if (data.aaData.length < data.iTotalRecords) {
-                $('#info-message').html("{% trans 'Showing the current page of data. Switch pages to see more data.' %}");
-                $('#report-info').removeClass('hide');
-            } else {
-                $('#report-info').addClass('hide');
+            if (mapSpec) {  // Only show warning for map reports
+                if (data.aaData !== undefined && data.iTotalRecords !== undefined) {
+                    if (data.aaData.length < data.iTotalRecords) {
```

Comment:
Looks like this is controlling a visibility toggle of `#report-info`. Should `#report-info` be hidden if `!mapSpec || data.aaData === undefined || data.iTotalRecords === undefined`, or is it intended to not modify `#report-info` in any way if that's true?